target/ppc/translate/vsx-impl.c.inc

```diff
@@ -2124,19 +2124,33 @@ static void gen_xvxsigdp(DisasContext *ctx)
     tcg_temp_free_i64(xbl);
 }
 
+/*
+ * Packed VSX Integer GER Flags
+ * 00 - no accumulation no saturation
+ * 01 - accumulate but no saturation
+ * 10 - no accumulation but with saturation
+ * 11 - accumulate with saturation
+ */
+static int pack_flags_xvi(int acc, int sat)
```

Comment:
We can also revert these changes and go back to using the enum options directly.